### DISCUSSION TITLE
error initializing the database: getting database schema version: unable to open database file

### DISCUSSION THREAD

**Discussion by Otamated** [stashapp/stash#4370] [Q&A]
I deleted all the config files and tried to "unistall" the software that way to start from fresh. And after installing the new version giving it new directories it gives me this error when setting up.

**Comment by WithoutPants** [comment]
We need a little more information than what has been provided.

What platform are you running on? Are you running using an exe/binary or via docker or something else? What configuration options did you select during setup?

**Comment by orion164** [comment]
Hi, I avoid opening a new discussion... maybe should I convert it to issue?
I have the same problem, I deleted stash and all the files to start from a fresh installation.

I am using Windows 10, Chrome.

I started the .exe file and started the configuration on chrome until after the directory selection step I receive the following error:

"error completing initialization: getting database schema version: unable to open database file"

I note that the error is received even if I do not set any folder and leave everything empty by default.

In the previous steps I selected all the available options by doing more tests but the error is always the same.

If I end the executable to restart it, it gives me the same error before opening the browser and to solve it I have to delete the .stash folder in the user folder.

Thanks, I hope this helps
